Mathlib/RingTheory/Polynomial/Content.lean

```diff
@@ -56,6 +56,15 @@ theorem IsPrimitive.ne_zero [Nontrivial R] {p : R[X]} (hp : p.IsPrimitive) : p 
 theorem isPrimitive_of_dvd {p q : R[X]} (hp : IsPrimitive p) (hq : q ∣ p) : IsPrimitive q :=
   fun a ha => isPrimitive_iff_isUnit_of_C_dvd.mp hp a (dvd_trans ha hq)
 
+theorem _root_.Irreducible.isPrimitive [NoZeroDivisors R]
+    {p : Polynomial R} (hp : Irreducible p) (hp' : p.natDegree ≠ 0) : p.IsPrimitive := by
```

Comment:
A few minor remarks:
- since the lemma is in the `_root_.Irreducible` namespace, it make sense to add a docstring that mentions polynomials;
- for the future: does it make sense to generalize `IsPrimitive` so that it works for `MvPolynomial`s too? E.g., generalize it to any algebra?
- for the not-so-distant future: please modify the module docstring and the docstring of `IsPrimitive` to mention that this notion has nothing to do with https://en.wikipedia.org/wiki/Primitive_polynomial_(field_theory)

Thanks!
bors d+

Reply:
For 1, 3: done in this PR.
For 2: Possibly? Though I don't have an immediate application yet. Maybe gauss' lemma could be generalized to multivariate polynomials and that might be it.

Reply:
Re 2: I just want to note that for an element in a free module over R, whether an element of R divides all coefficients is independent of the basis chosen.